### PR TITLE
#71 - Use BasicAuthSlice, extract test classes

### PR DESCRIPTION
**Pull request by genryxy**
Part of #71 
Used `BasicAuthSlice`, extracted test classes

### CODE REVIEW

File: src/main/java/com/artipie/composer/http/PhpComposer.java [hunk 1]
```diff
@@ -37,13 +42,16 @@
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
 public final class PhpComposer extends Slice.Wrap {
-
     /**
-     * Ctor.
-     *
-     * @param repository Repository.
+     * Ctor with existing front and default parameters for free access.
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/composer/http/PhpComposer.java:46]: @genryxy javadoc seems off, no default parameters in this ctor.

**Review by olenagerasimova — APPROVED** [review]
@genryxy just one minor comment, please, take a look

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
sorry, wrong button last time)

**Comment by genryxy** [comment]
@olenagerasimova thanks, corrected

**Review by olenagerasimova — APPROVED** [review]
@genryxy thanks

**Comment by 0crat** [comment]
Job `gh:artipie/composer-adapter#81` is not assigned, can't get performer

**Pull request merged**